nncf/quantization/algorithms/bias_correction/algorithm.py

```diff
@@ -484,10 +486,11 @@ def output_filter_func(point):
             output_fp.extend(tensor_collector.get_statistics().mean_values)
         return np.array(output_fp)
 
-    def get_statistic_points(self, model: TModel) -> StatisticPointsContainer:
+    def get_statistic_points(self, model: TModel, graph: NNCFGraph) -> StatisticPointsContainer:
```

Comment:
Good question! @KodiaqQ, could you answer on this comment.

Reply:
To be true, the model here, for BC, is without quantizers, and `remove_fq_from_inputs` may be removed. But let's keep it for the next PRs because BC algo is not the main part of DQ.